src/components/picker/index.js

```diff
@@ -186,6 +186,10 @@ class Picker extends Component {
           value: nextProps.value
         };
       }
+    } else if (_.isFunction(nextProps.renderPicker) && prevState.value !== nextProps.value) {
+      return {
+        value: nextProps.value
```

Comment:
Don't you need to update `prevValue` as well?

Reply:
I don't think so, not sure what `prevValue` is used for, it seems to reside only in the previous `if` statement.

Reply:
We store `prevValue` on state for the `getDerivedStateFromProps` comparison. I think it has to be updated every time we update `value` otherwise the condition won't be up-to-date...

Reply:
ok, updated